README.md

```diff
@@ -8,7 +8,7 @@ Usage:
 Install with `npm install react-absolute-grid`
 
     import React from 'react';
-    import AbsoluteGrid from './lib/AbsoluteGrid.jsx';
+    import AbsoluteGrid from './lib/AbsoluteGrid.js';
```

Comment:
why not change readme to reflect how people will use in real world

`import AbsoluteGrid from 'react-absolute-grid;'`